PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetPerformance.cxx

```diff
@@ -2190,7 +2228,30 @@ void AliAnalysisTaskEmcalJetPerformance::ComputeBackground()
   AliJetContainer* jetCont = 0;
   TIter nextJetColl(&fJetCollArray);
   while ((jetCont = static_cast<AliJetContainer*>(nextJetColl()))) {
-    
+	  if(jetCont->GetNJets()<1)continue;//Don't enter if there are no jets in the container
```

Comment:
Please also in general follow the same coding conventions as in the rest of the task (spacing, brackets for if statements)

Reply:
you have also used such statements, like 1397 or 1505 btw. I have added now the bracket.